src/components/metadata/index.js

```diff
@@ -109,6 +125,11 @@ const MetaData = ({ visible = true, metadata, onToggleNodeSelected }) => {
             )}
           </div>
         </MetaDataRow>
+        <MetaDataRow
+          label="Description (docstring):"
+          visible={isTaskNode}
+          value={metadata.docstring}
+        />
```

Comment:
![image](https://user-images.githubusercontent.com/1155816/101910621-18705100-3bb7-11eb-9dac-6bf0b8f5fb17.png)

Additional question: If we are going to always show these fields regardless of whether they're populated, as long as the node is the right corresponding type, then can we we please add dummy data to the demo dataset as well? We want the demo to always look as good as possible for obvious reasons.

Also, potentially the animals one as well? What do you think?

This can be part of a separate PR if you prefer.

Reply:
Yeah for sure the included data sets should be updated with the new fields. I was planning on getting your input on how to do that, as now we have two API calls, I wasn't sure how best to implement that. I think we should do it in a separate PR though from the functionality.

Reply:
ok cool. yeah I think for the demo dataset we can definitely inline it. Once we add the code snippets it'll get trickier and we may need to revisit this whole data loading thing from the ground up